FppTest/port/build-exe

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
```

Comment:
Is this file left over from something?  Shouldn't these items be done in the CMake system?

Reply:
These scripts are to work around issues in the code coverage analysis. I just opened an issue on this: https://github.com/nasa/fprime/issues/1862.

As discussed let's leave these scripts in so we have a way to generate code coverage until #1862 is resolved. Also the scripts may help point the way to resolving #1862.